march_simulation/config/effort_control/march4.yaml

```diff
@@ -35,6 +35,7 @@ march:
       stop_trajectory_duration: 0        # Override default
 
       constraints:
+        goal_time: 0.1
```

Comment:
Whats this?

Reply:
I thought it is the max allowed error in the time at which you reach your goal. Otherwise a FollowJointTrajectory GoalError is thrown.